features/interaction.yml

```
@@ -0,0 +1,12 @@
+name: Interaction media queries
+description: "The `pointer`, `any-pointer`, `hover`, and `any-hover` CSS media queries set styles based on the presence of pointing devices and their ability to hover over elements. For example, a smartphone device would match the `(hover: none) and (pointer: coarse)` media query."
```

I think "would" makes this more abstract that it needs to be. What do you think of this?

```suggestion
description: "The `pointer`, `any-pointer`, `hover`, and `any-hover` CSS media queries set styles based on the presence of pointing devices and their ability to hover over elements. For example, most smartphones match the `(hover: none) and (pointer: coarse)` media query."
```